### DISCUSSION TITLE
QUESTION｜对于PEP8命名规范的建议

### DISCUSSION THREAD

**Discussion by shenjackyuanjie** [MCSLTeam/MCSL2#83] [提问区域]
参考PEP8
我已经预见到如果我跑一遍ruff，会是多么惨烈的结果
请勿用C#的命名规范去写Python啊啊啊啊啊（

**Comment by lxhtt** [comment]
命名规范见鬼去（
主要是我这样写习惯了，太难改了

**Comment by shenjackyuanjie** [comment]
主要是，咱可以改一下非生成部分的规范吧
![image](https://user-images.githubusercontent.com/54507071/226654935-7fa56e4e-3bed-4e07-a1f0-68bd376fdec4.png)
比如Java Detector
https://github.com/LxHTT/MCSL2/blob/master/MCSL2_JavaDetector.py

**Comment by shenjackyuanjie** [comment]
算了（
就这样吧（
